book/forms.rst

```diff
@@ -77,8 +77,8 @@ from inside a controller::
     // src/AppBundle/Controller/DefaultController.php
     namespace AppBundle\Controller;
 
-    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
     use AppBundle\Entity\Task;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
```

Comment:
We don't have a convention I think (the code doesn't have one either, does it? /cc @stof )

I always sort on length of the seperate namespace items, but I don't care much :)